src/test/java/seedu/clinic/logic/commands/CommandTestUtil.java

```diff
@@ -75,31 +76,47 @@
     public static final String VALID_TAG_FEVER = "fever";
     public static final String VALID_TAG_ANTIBIOTICS = "antibiotics";
     public static final String VALID_TAG_PAINKILLER = "painkiller";
+    public static final String VALID_TYPE_SUPPLIER = "s";
+    public static final String VALID_TYPE_WAREHOUSE = "w";
```

Comment:
Here we can actually use the enum class Type.supplier and Type.wareshouse.

Reply:
Oh yea I tried to use Type.supplier and Type.wareshouse but ran into some difficulty when testing for invalid input. I would have to create false input of Type format to do testing. I'll look into this again to see how we can resolve it, same as for the one below.